frontend/src/setupTests.tsx

```diff
@@ -42,6 +43,11 @@ jest.mock('@react-pdf/renderer', () => ({
   Font: { register: () => {} },
 }));
 
+// https://github.com/remarkjs/react-markdown/issues/635
+jest.mock('react-markdown', () => (props: { children: React.ReactNode }) => {
+  return <>{props.children}</>;
+});
```

Comment:
This is unfortunately, I would have rather rendered this properly in our jest tests. However, the alternative approach seemed more frustrating to maintain than the value it would provide. 

See: https://github.com/remarkjs/react-markdown/issues/635

Example alternative: 
<img width="728" alt="Screenshot 2024-06-18 at 10 00 54 AM" src="https://github.com/WFP-VAM/prism-app/assets/8203830/81d04f5f-2498-4add-8b87-10e3277c2819">